modules/402-ingress-nginx/monitoring/prometheus-rules/ingress-nginx.yaml

```diff
@@ -169,3 +169,15 @@
         3. If the `Number of Nodes Scheduled with Up-to-date Pods` parameter does not match
         `Current Number of Nodes Scheduled`, check the pertinent Ingress Nginx Controller's 'nodeSelector' and 'toleration' settings,
         and compare them to the relevant nodes' 'labels' and 'taints' settings
+  - alert: NginxIngressDeprecatedVersion
+    expr: d8_ingress_nginx_controller{controller_version="1.1"}
```

Comment:
could you please update the expression to something like:
`count(d8_ingress_nginx_controller{controller_version="1.1"}) > 0`, otherwise we risk end up having more than one alert in case there are several ingress nginx controllers of `1.1` in the cluster.
![image](https://github.com/deckhouse/deckhouse/assets/41360396/673c9823-d192-4d5a-8ab3-295207d5988d)